.pylintrc

```diff
@@ -251,7 +251,7 @@ expected-line-ending-format=
 # Logging modules to check that the string format arguments are in logging
 # function parameter format
 logging-modules=logging
-
+disable=logging-fstring-interpolation
```

Comment:
```suggestion
```

Using f-strings with logging is not considered safe because if the string interpolation fails nothing gets logged. if you use the old-school string interpolation then the message always get logged, even if the impersonation fails.